CoroutineTemplate/README.md

```diff
@@ -0,0 +1,25 @@
+# Android Templates: Kotlin
```

Comment:
This template is not focused on Kotlin, should it be Coroutine?

Reply:
Might need to change this in `RxJavaTemplate` as well 🙏